### DISCUSSION TITLE
.NET Framework 4.8 - Graph 5.56 - System.IO.FileNotFoundException: 'Could not load file or assembly 'System.Runtime, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.'

### DISCUSSION THREAD

**Discussion by SebMANE** [microsoftgraph/msgraph-sdk-dotnet#2589] [Q&A]
Hello, 

I was using Graph SDK (1.16) in a SharePoint full trust solution based on .NET Framwork 4.6.1 since a long time without issue.

I need to upgrade to a recent version to use SignInActivity but I'm stuck with this error (reproduced in a .NET framework 4.8 console application):

System.IO.FileNotFoundException: 'Could not load file or assembly 'System.Runtime, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The system cannot find the file specified.'

It looks like the latest package for System.Runtime is 4.3.1 : https://www.nuget.org/packages/System.Runtime 

Do you have any idea ?

**Comment by andrueastman** [comment]
Any chance you're able to share the dotnet SDK versions installed in your machine? Do you have .NET framework 4.8?

**Reply by SebMANE** [reply]
Yes we have several .NET framework installed (including 4.8), it's a SharePoint Server 2019 development environment. I can reproduce the issue with different ones (4.6.1, 4.7.2, 4.8).

After many tests I figured out that the problem comes from Graph library 5.56.0.

With the previous version 5.55.0 it's OK, on 4.7.2 or 4.8 frameworks with a fresh new console app

**Reply by andrueastman** [reply]
Any chance you can share a sample console app that fails to build? I'm unable to replicate this from our end. 
Versions `5.55.0` and `5.56.0` both share the same dependencies as well..
